### ISSUE TITLE
Mermaid diagrams not rendering for multiple documentation pages

### ISSUE THREAD

**Issue by McPatate** [kubernetes/website#37357]
All graphs on this doc page are showing a Mermaid error, as follows:

![Screenshot from 2022-10-18 11-58-55](https://user-images.githubusercontent.com/9112841/196399642-8573bc41-c4cf-4d4e-a330-cd9ac8174359.png)

**Comment by utkarsh-singh1** [comment]
Hi @McPatate, can you share the link of the doc where the error is showing?

**Comment by McPatate** [comment]
This was the page, though I see it working on my mobile now:
https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/

I'll check on my laptop tomorrow to see if this is still an issue.

**Comment by sftim** [comment]
I can't replicate this on my computer. It'd be good to know some reliable steps-to-replicate including what OS, what CPU architecture, and what software.

**Comment by sftim** [comment]
/area web-development

For now:
/priority awaiting-more-evidence

**Comment by McPatate** [comment]
Well, I have graphs again, even on the computer it wasn't working on before. No idea what went wrong here, sorry !

I'll let you close the issue.

**Comment by utkarsh-singh1** [comment]
Hi @McPatate, thanks for your update, I have also checked on my device it seems to be working. It seems it was an error from 3rd party tool mermaid, I am running on intel x86_64 and fedora OS and this error seems to be frequent.

**Comment by McPatate** [comment]
Firefox, intel x86_64, popOS on my end :)

**Comment by sftim** [comment]
I can replicate this issue :disappointed: 

/triage accepted
/remove-priority awaiting-more-evidence

**Comment by sftim** [comment]
/retitle Mermaid diagrams not rendering for “Pod Topology Spread Constraints” concept

**Comment by chrismetz09** [comment]
Cannot replicate using Chrome and FF. Indicates a parser error or sometimes Mermaid can't pick up the configuration if mermaid.js is loaded from cache.

Investigating further.

**Comment by hujun-open** [comment]
have this issue with FF on windows 10; 
chrome works fine though

**Comment by utkarsh-singh1** [comment]
Mostly, I have faced this issue on both Firefox and chrome when network connection is relatively slow.

**Comment by SergiyKolesnikov** [comment]
After reloading the page, the diagrams are displayed correctly. Firefox 107.0.1 (64-bit), Debian 11.

**Comment by sftim** [comment]
/retitle Mermaid diagrams not rendering for multiple documentation pages

**Comment by shurup** [comment]
I can confirm this issue on my Firefox/Linux, too. It appears when I open this page for the first time. After I do a refresh (F5), this issue disappears and I can see the actual charts.

While investigating this issue, I found https://github.com/squidfunk/mkdocs-material/issues/3232 and — funny enough — there I found a reference to #31960 with the same investigation. Shouldn't we close all duplicated issues regarding broken Mermaid diagrams?
